### ISSUE TITLE
Mysql version 5.6

### ISSUE THREAD

**Issue by dimasdwika** [phansible/phansible#259]
I think now modern technology requires minimum Mysql version 5.6. At the moment, after downloading from Phansible, there's no way to tell on ansible to install mysql 5.5 or 5.6. By ubuntu 14.04 default when you install mysql it will install mysql 5.5 not 5.6.

I think it should be able to choose MYSQL version need to be installed (like the one in PHP).

Cheers,
Dimas

**Comment by nickmartyn** [comment]
Hi,  
try to change in /roles/mysql/tasks/main.yml to this:  
<pre>
- name: mysql | Install MySQL Packages  
  sudo: yes  
  apt: pkg={{ item }} state=latest  
  with_items:  
    - mysql-server-5.6 
    - mysql-client-5.6
    - python-mysqldb 
</pre>
And increase your virtual machine memory to 1024 mb ram or higher (mysql server 5.6 requires more memory)

**Comment by naxhh** [comment]
Good point @nickmartyn!
That should work.

We are open to features and being able to choose mysql version seems a decent one to me.
If you want to try to contribute tell me and I can provide you some guidelines.

**Comment by dimasdwika** [comment]
yeah, that would be lovely if we can change from the UI phansible. I would try to fork the repo and do it from there and request PR.

**Comment by naxhh** [comment]
nice! anything you need just ask here or on gitter

**Comment by debo** [comment]
@dimasdwika do you have any update on this or would you like me to have a look?
